index.bs

```diff
@@ -255,96 +255,84 @@ Processing model {#sec-processing-model}
-                    1. Set |attribution|'s {{containerSrc}} attribute to the value of |container|'s <{object/data}> content attribute, or the empty string if the attribute is absent.
-                1. If |container| is an <{embed}> element:
-                    1. Set |attribution|'s {{containerType}} attribute to "<code>embed</code>".
-                    1. Set |attribution|'s {{containerName}} attribute to the empty string.
-                    1. Set |attribution|'s {{containerSrc}} attribute to the value of |container|'s <{embed/src}> content attribute, or the empty string if the attribute is absent.
-
-        1. Create a new {{PerformanceLongTaskTiming}} object |newEntry| with |destinationRealm| and set its attributes as follows:
-
-            1. Set |newEntry|'s {{PerformanceEntry/name}} attribute to |name|.
-            1. Set |newEntry|'s {{PerformanceEntry/entryType}} attribute to "<code>longtask</code>".
-            1. Set |newEntry|'s {{PerformanceEntry/startTime}} attribute to |start time|.
-            1. Set |newEntry|'s {{PerformanceEntry/duration}} attribute to |end time| minus |start time|.
-            1. If |attribution| is not <code>null</code>, set |newEntry|'s {{PerformanceLongTaskTiming/attribution}} attribute to a new frozen array containing the single value |attribution|.
-
-                NOTE: future iterations of this API will add more values to the {{PerformanceLongTaskTiming/attribution}} attribute, but for now it only contains a single value.
-
-        1. <a>Queue the PerformanceEntry</a> |newEntry|.
+    1. Let |destinationRealm| be |document|'s <a>relevant Realm</a>.
+
+    1. Let |name| be the empty string. This will be used to report <a>minimal culprit attribution</a>, below.
+    1. Let |culpritSettings| be <code>null</code>.
+    1. Process |task|'s <a>script evaluation environment settings object set</a> to determine |name| and |culpritSettings| as follows:
+
+        1. If |task|'s <a>script evaluation environment settings object set</a> is empty: set |name| to "<code><a>unknown</a></code>" and |culpritSettings| to <code>null</code>.
+        1. If |task|'s <a>script evaluation environment settings object set</a>'s length is greater than one: set |name| to "<code><a>multiple-contexts</a></code>" and |culpritSettings| to <code>null</code>.
+        1. If |task|'s <a>script evaluation environment settings object set</a>'s length is one:
+            1. Set |culpritSettings| to the single item in |task|'s <a>script evaluation environment settings object set</a>.
+            1. Let |destinationSettings| be |destinationRealm|'s <a>relevant settings object</a>.
+            1. Let |destinationOrigin| be |destinationSettings|'s [=environment settings object/origin=].
+            1. Let |destinationBC| be |destinationSettings|'s <a>responsible browsing context</a>.
```

Comment:
We recently removed "responsible browsing context". Instead you can use |destinationSetting|'s [global object](https://html.spec.whatwg.org/#concept-settings-object-global)'s [browsing context](https://html.spec.whatwg.org/#window-bc). 

Separately, you might need to check if that's null. (This was a preexisting issue, but made more obvious by the new architecture.) That can happen if the realm corresponds to a removed iframe.

These same remarks apply to all other uses of "responsible browsing context". Sorry; I didn't realize you used the term so much, or I would have sent a PR.

Reply:
Done